demo/casc/Jenkinsfile

```diff
@@ -0,0 +1,11 @@
+pipeline {
```

Comment:
Add a small README which describes how to run the demo?

Reply:
I added a README and included a reference to it in the Docker guide. Let me know what you think :)